src/components/pages/Invite/Invite.tsx

```diff
@@ -16,7 +16,10 @@ import { RahaState } from "../../../store";
 import { generateToken } from "../../../helpers/token";
 import { SendInvite } from "./SendInvite";
 
+const ENABLE_SEND_INVITE = false;
+
 enum InviteStep {
+  WIP,
```

Comment:
What is the meaning of the WIP step? Work in Progress doesn't really make sense to me

Reply:
It means that Invite flow is WIP. It's temporary anyway